docs/manuals/userman/sdm/events/index.md

```diff
@@ -0,0 +1,39 @@
+This page covers how you can utilize the Core event consumer feature to get notifications
+about the changes to the entities in the Core service.
+
+Objects of class `Event^` are used to represent the changes in the Core service.
+An `Event^` object holds the necessary attributes to identify the change.
+
+For more details and examples, see the
+[event descriptions](../events-description.md) page.
+
+`CoreEventConsumerBase^` provides a framework for defining custom logic to consume `Event^` from
+a queue in a separate thread
```

Comment:
```suggestion
In this section, we explore how to register and process *events*.

# What is an Event

Taipy, particularly the scenario and management capability, is designed to be natively
multiuser and asynchronous. That means you cannot control when an action is done or completed
(an end-user created a scenario, my submission failed, a data node has been edited, a job
completed, etc.). For that purpose, Taipy emits *events* that you can consume and react to.

An *event* is a message that Taipy emits when something happens in the system. It is
represented by an object of class `Event^`. An *event* holds the necessary attributes to
identify the change.

!!! example "Scenario creation event"

    For example, when a scenario is created, an event is emitted with
    the following attributes:

        - `entity_type`: SCENARIO
        - `operation`: CREATION
        - `entity_id`: the id of the scenario
        - `creation_date`: the date and time of the event creation

Events are particularly useful when you want to:

- Update the user interface (e.g., update a list of scenarios when a new one is created)
- Trigger an action (e.g., automatically submit a scenario when its input data is updated)
- Notify end-users (e.g., send a GUI notification to all users when a job fails)
- etc.

For more details, see the [event](events-description.md) page.

# What is a Registration

Taipy exposes the `Notifier.register()^` method to register to events. When you register,
you specify parameters that define the events you want to receive. These parameters
allow you to filter the events you want to receive.
For more details, see the [registration](understanding-topics.md) page.
```

Reply:
I agree, I tried to push more info to other pages like event and topic but it probably doesn't make it clearer